### DISCUSSION TITLE
How to visualize when training like isaacGYM

### DISCUSSION THREAD

**Discussion by kassasin** [google/brax#563] [General]
I want to visualize training process (not only rewards but also robot moving), does brax suport this>

**Comment by btaba** [comment]
Hi @kassasin , you can use the `policy_params_fn` [callback](https://github.com/google/brax/blob/417465cbd52740bddedd4b3ba89ee941d90f9124/brax/training/agents/ppo/train.py#L157C5-L157C21) to render videos for every eval that gets run

**Reply by kassasin** [reply]
Thanks，I will take a try .

**Comment by kassasin** [comment]
I think there have some differences . I can use "policy_params_fn" to save policy and use it to visualize some robot, but I think it's not necessary. We can use every enviroment's robot state to visualize .Is my understanding correct? @btaba
